platform/javascript/detect.py

```diff
@@ -213,6 +212,9 @@ def configure(env):
     # Reduce code size by generating less support code (e.g. skip NodeJS support).
     env.Append(LINKFLAGS=["-s", "ENVIRONMENT=web,worker"])
 
+    # Break Chrome for android? Need to report upstream.
+    env.Append(LINKFLAGS=["-s", "BINARYEN_EXTRA_PASSES=--one-caller-inline-max-function-size=1"])
```

Comment:
In https://github.com/emscripten-core/emscripten/issues/13899 they tested different values and it seems all users had performance regressions with 19307 and not 19306. Maybe forcing it to 1 is a bit overkill? Should we try something safe-ish like 4096 or 8192?

Reply:
I did some tests and found 2048 was working on my phone while results with 4096 were a bit inconsistent.
We can use that, but maybe we should test it on some (low-end) devices?

Reply:
For reference, the upstream issue could be this: https://bugs.chromium.org/p/v8/issues/detail?id=11863

Reply:
So, I've loaded some tests here (Chrome for Android `91.0.4472.101`):
- [2048](https://no-war.fales.me/droid/2048/): works
- [4096](https://no-war.fales.me/droid/4096/): fails
- [8192](https://no-war.fales.me/droid/8192/): fails

We could alternatively wait a little bit more and see it the upstream bug gets fixed instead.

Reply:
Maybe we can play it safe with `512` or something, and link the upstream report in a comment, so we know to revert or increase it once the bug is fixed.